src/style/colors.ts

```diff
@@ -211,7 +211,10 @@ function validateHex(value: string) {
 }
 
 function threeDigitHexToSix(value: string) { 
-  return value.replace(/./g, '$&$&');
+  const firstDigit = value[0];
+  const secondDigit = value[1];
+  const thirdDigit = value[2];
+  return firstDigit.concat(firstDigit, secondDigit, secondDigit, thirdDigit, thirdDigit);
```

Comment:
Why not just return `value + value` ?

Reply:
We need to double each letter: 
`threeDigitHexToSix(123)` should return `112233` and not `123123`

Reply:
You are completely right! my bad :( 
In that case the regex solution is definitely preferable... 
apologies for the trouble